Mastodon/Extension/UICollectionViewDiffableDataSource.swift

```diff
@@ -13,28 +13,14 @@ extension UICollectionViewDiffableDataSource {
         snapshot: NSDiffableDataSourceSnapshot<SectionIdentifierType, ItemIdentifierType>,
         completion: (() -> Void)? = nil
     ) {
-        if #available(iOS 15.0, *) {
-            self.applySnapshotUsingReloadData(snapshot, completion: completion)
-        } else {
-            self.apply(snapshot, animatingDifferences: false, completion: completion)
-        }
+        self.applySnapshotUsingReloadData(snapshot, completion: completion)
     }
     
     func applySnapshot(
```

Comment:
I think these functions can be dropped entirely in favor of just calling `apply(snapshot, ...)`, right?

Reply:
You are absolutely right 👍 
Great, that you saw that :)